src/main/resources/public/css/styles.css

```diff
@@ -196,6 +196,60 @@ background-color: transparent;
     position:absolute;
 }
 
+.infoBox {
+    display: block;
+    width: 300px;
+    padding: 10px;
+    vertical-align: top;
+}
+
+.shipIconDivCol1 {
+    display: inline-block;
+    position: relative;
+    vertical-align: top;
+}
+
+.shipIconBox {
+    display: block;
+    vertical-align: top;
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    overflow: hidden;
+    padding: 15px;
+    position: relative;
+}
+
+.shipIconBox2 {
```

Comment:
You might need to try adjusting this name to be a little more descriptive. I'm not quite sure what you are referencing with shipIconBox